release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -3747,3 +3747,27 @@ $ oc adm release info 4.11.49 --pullspecs
 ==== Updating
 
 To update an existing {product-title} 4.11 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-11-50"]
+=== RHBA-2023:5350 {product-title} 4.11.50 bug fix update
+
+Issued: 2023-09-26
+
+{product-title} release 4.11.50 is now available. Bug fixes included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2023:5350[RHBA-2023:5350] advisory. RPM packages included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2023:5352[RHBA-2023:5352] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.11.50 --pullspecs
+----
+
```

Comment:
This needs to be included like in 4.13 and 4.12 https://issues.redhat.com/browse/OCPBUGS-18545 as a feature write up that is a backport. [This](https://docs.openshift.com/container-platform/4.13/release_notes/ocp-4-13-release-notes.html#ocp-4-13-12-custom-rhcos-image-for-gcp) content needs to be added to this RN.